applications/luci-app-smartdns/root/usr/share/rpcd/acl.d/luci-app-smartdns.json

```diff
@@ -0,0 +1,22 @@
+{
+	"luci-app-smartdns": {
+		"description": "Grant access to LuCI app smartdns",
+		"read": {
+			"file": {
+				"/etc/smartdns/*": [ "read" ],
+				"/usr/sbin/iptables": [ "exec" ],
+				"/usr/sbin/ip6tables": [ "exec" ],
```

Comment:
Please change this line to:
`"/usr/sbin/ip6tables -t nat -nL PREROUTING": [ "exec" ],`